LayoutTests/imported/w3c/web-platform-tests/css/cssom/cssom-getPropertyValue-common-checks-expected.txt

```diff
@@ -9,5 +9,5 @@ PASS All aliases can serialize target property set to its initial value
 PASS Can't serialize shorthand when longhands are set to different css-wide keywords
 PASS Can't serialize shorthand when longhands have different priority
 PASS Can't serialize shorthand set to 'initial' when some longhand is missing
-FAIL Can't serialize shorthand set to initial value when some longhand is missing assert_array_equals: lengths differ, expected array [] length 0, got […, ["font", "line-height", "16px -webkit-standard"], ["font", "font-size-adjust", "16px -webkit-standard"], ["font", "font-kerning", "16px -webkit-standard"], ["font", "font-variant-alternates", "16px -webkit-standard"], ["font", "font-variant-ligatures", "16px -webkit-standard"], ["font", "font-variant-numeric", "16px -webkit-standard"], ["font", "font-variant-east-asian", "16px -webkit-standard"], ["font", "font-variant-position", "16px -webkit-standard"], ["font", "font-feature-settings", "16px -webkit-standard"], ["font", "font-optical-sizing", "16px -webkit-standard"], ["font", "font-variation-settings", "16px -webkit-standard"], ["font", "font-palette", "16px -webkit-standard"], ["font-synthesis", "font-synthesis-weight", "style small-caps"], ["font-synthesis", "font-synthesis-style", "weight small-caps"], ["font-synthesis", "font-synthesis-small-caps", "weight style"], ["offset", "offset-path", "auto"], ["offset", "offset-distance", "auto none"], ["offset", "offset-position", "none"], ["offset", "offset-anchor", "auto none"], ["offset", "offset-rotate", "auto none"]] length 24
+FAIL Can't serialize shorthand set to initial value when some longhand is missing assert_array_equals: lengths differ, expected array [] length 0, got […, ["font", "line-height", "16px -webkit-standard"], ["font", "font-size-adjust", "16px -webkit-standard"], ["font", "font-kerning", "16px -webkit-standard"], ["font", "font-variant-alternates", "16px -webkit-standard"], ["font", "font-variant-ligatures", "16px -webkit-standard"], ["font", "font-variant-numeric", "16px -webkit-standard"], ["font", "font-variant-east-asian", "16px -webkit-standard"], ["font", "font-variant-position", "16px -webkit-standard"], ["font", "font-feature-settings", "16px -webkit-standard"], ["font", "font-optical-sizing", "16px -webkit-standard"], ["font", "font-variation-settings", "16px -webkit-standard"], ["font", "font-palette", "16px -webkit-standard"], ["font-synthesis", "font-synthesis-weight", "style small-caps"], ["font-synthesis", "font-synthesis-style", "weight small-caps"], ["font-synthesis", "font-synthesis-small-caps", "weight style"], ["offset", "offset-path", "auto"], ["offset", "offset-distance", "auto none"], ["offset", "offset-position", "none"], ["offset", "offset-anchor", "auto none"], ["offset", "offset-rotate", "auto none"]] length 29
```

Comment:
Why is this increasing?

Reply:
Because more longhands that are simply reset to their initial values are now listed here. It’s no mystery and really not important. Eventually I will fix this!

Reply:
After I fixed some more things, this went away. So it was probably a bug.